libpermian/cli/__init__.py

```diff
@@ -28,6 +29,11 @@ def main(*raw_args):
     Custom CLI arguments can be optionally provided as \*args. If those are not
     provided sys.argv content is used as CLI arguments.
     """
+    # Set pseudo-terminal as fd 0 (stdin), this breaks input into pipeline but
```

Comment:
I just realized it could be also worth doing this only if 0 is not a tty, like:
```python
if not os.isatty(0):
    ...
```
https://docs.python.org/3/library/os.html#os.isatty

Reply:
Good idea, thanks